corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -403,19 +406,34 @@ def _row_to_row_elem(row, index, is_total_row=False):
                     filter_options_by_field[k].add(value)
             return row_elem
 
+        total_column_ids = data_source.get_total_column_ids()
+        total_cols = {col_id: 0 for col_id in total_column_ids}
+
+        def _update_total_row(row):
+            for col_id in total_column_ids:
+                val = row[col_id]
+                if isinstance(val, numbers.Number):
+                    total_cols[col_id] += val
```

Comment:
This would only work where `aggregation = sum` in the report right? What about mins, maxs, and averages?

Reply:
interesting. that seems kind of wierd to me, but it would make this much easier to maintain